### ISSUE TITLE
Permanent dot shown on Firefox asking for "Permission needed"

### ISSUE THREAD

**Issue by fcastilloec** [pip8786/CCNA#22]
This issue has been going on for a while. The dot is added by Firefox under the extension to signal the user that the extension needs attention. The extension is working without any issues, and when checking the permission tab for this extension, all websites have been allowed.
The dot disappears after interacting with the extension, but loading a new tab will make it come back.
This is not the only extension that has this problem, something probably change with the way permissions are asked, or with the new manifest. This will also explain why loading a new tab triggers this bug.

**Comment by fcastilloec** [comment]
I also found this bug report at Mozilla:
[Manifest V3 extensions with low privilege activeTab shows annoying blue dot for all websites](https://bugzilla.mozilla.org/show_bug.cgi?id=1851083)
There are some workarounds mentioned there, from asking for more permissions to using manifest v2. So keeping a v2 for Firefox and v3 for Chrome will probably be the best option until they come up with something better

**Comment by pip8786** [comment]
@fcastilloec is this still the case?
@MM-John do you also see this dot?

**Comment by fcastilloec** [comment]
The truth is that I've stopped using your extension a while ago, the dot was annoying me too much. I've also switched browsers since then. Nonetheless, I just started my Firefox, and your extension doesn't show any dots anymore. You can close this issue if you'd like

**Comment by MM-John** [comment]
No, I do not see the "needs attention" dot on this extension. All looks well here.
